dashboard/app/helpers/lesson_import_helper.rb

```diff
@@ -0,0 +1,213 @@
+# Helper module for importing data from curriculumbuilder
+module LessonImportHelper
+  # Lockable lessons don't need to be merged with curriculumbuilder, but we do
+  # need the levels to be part of an activity section.
+  def self.update_lockable_lesson(levels, lesson_id)
+    lesson_activity = LessonActivity.new
+    lesson_activity.lesson_id = lesson_id
+    lesson_activity.seeding_key = SecureRandom.uuid
+    lesson_activity.position = 1
+    activity_section = ActivitySection.new
+    activity_section.seeding_key = SecureRandom.uuid
+    activity_section.position = 1
+    activity_section.lesson_activity = lesson_activity
+    activity_section.save!
```

Comment:
do you need to call lesson_activity.save! too?

Reply:
I think `lesson_activity.save` is automatically called when it's set as the lesson_activity of the activity_section. Not 100% sure though...